html/en/beats/libbeat/master/beats-reference.html
html/en/beats/libbeat/master/breaking-changes-7.0.html
--- /dev/fd/63
+++ html/en/beats/libbeat/master/breaking-changes-7.0.html
@@ -278,7 +278,7 @@
            </a>
           </h4>
           <div class="table">
-           <a id="id-1.8.8.16">
+           <a id="id-1.8.8.13">
            </a>
            <p class="title">
             <strong>
@@ -931,7 +931,7 @@
            </div>
           </div>
           <div class="table">
-           <a id="id-1.8.8.17">
+           <a id="id-1.8.8.14">
            </a>
            <p class="title">
             <strong>
@@ -5664,7 +5664,7 @@
            </div>
           </div>
           <div class="table">
-           <a id="id-1.8.8.18">
+           <a id="id-1.8.8.15">
            </a>
            <p class="title">
             <strong>
@@ -6029,7 +6029,7 @@
            </div>
           </div>
           <div class="table">
-           <a id="id-1.8.8.19">
+           <a id="id-1.8.8.16">
            </a>
            <p class="title">
             <strong>
@@ -6346,7 +6346,7 @@
            </div>
           </div>
           <div class="table">
-           <a id="id-1.8.8.20">
+           <a id="id-1.8.8.17">
            </a>
            <p class="title">
             <strong>
@@ -7367,7 +7367,7 @@
            </div>
           </div>
           <div class="table">
-           <a id="id-1.8.8.21">
+           <a id="id-1.8.8.18">
            </a>
            <p class="title">
             <strong>
@@ -8068,7 +8068,7 @@
            </div>
           </div>
           <div class="table">
-           <a id="id-1.8.8.22">
+           <a id="id-1.8.8.19">
            </a>
            <p class="title">
             <strong>
@@ -8700,7 +8700,7 @@
            The Auditbeat JSON data types produced by the output have been changed to align with the data types used in the Elasticsearch index template.
           </p>
           <div class="table">
-           <a id="id-1.8.8.26">
+           <a id="id-1.8.8.22">
            </a>
            <p class="title">
             <strong>
html/en/beats/libbeat/master/breaking-changes-7.1.html
html/en/beats/libbeat/master/breaking-changes-7.2.html
html/en/beats/libbeat/master/breaking-changes-7.3.html
html/en/beats/libbeat/master/breaking-changes.html
html/en/beats/libbeat/master/community-beats.html
html/en/beats/libbeat/master/config-file-format-cli.html
html/en/beats/libbeat/master/config-file-format-env-vars.html
html/en/beats/libbeat/master/config-file-format-namespacing.html
--- /dev/fd/63
+++ html/en/beats/libbeat/master/config-file-format-namespacing.html
@@ -180,7 +180,7 @@
           <p>
            gets collapsed into
            <code class="literal">
-            output.elasticsearch.index: 'beat-%{[agent.version]}-%{+yyyy.MM.dd}'
+            output.elasticsearch.index: 'beat-%{[agent.version]}-%{+yyyy.MM.dd}\'
            </code>
            . The full name of a setting is based on all parent structures involved.
           </p>
html/en/beats/libbeat/master/config-file-format-tips.html
html/en/beats/libbeat/master/config-file-format-type.html
html/en/beats/libbeat/master/config-file-format.html
--- /dev/fd/63
+++ html/en/beats/libbeat/master/config-file-format.html
@@ -219,7 +219,7 @@
            for lists:
           </p>
           <div class="pre_wrapper lang-yaml">
-           <pre class="programlisting prettyprint lang-yaml">person: {name: "John Doe", age: 34, country: "Canada"}
+           <pre class="programlisting prettyprint lang-yaml">person: \{name: "John Doe", age: 34, country: "Canada"}
 colors: ["Red", "Green", "Blue"]</pre>
           </div>
           <p>
html/en/beats/libbeat/master/config-file-permissions.html
html/en/beats/libbeat/master/config-gile-format-refs.html
--- /dev/fd/63
+++ html/en/beats/libbeat/master/config-gile-format-refs.html
@@ -177,7 +177,7 @@
            For example the filebeat registry file defaults to:
           </p>
           <div class="pre_wrapper lang-yaml">
-           <pre class="programlisting prettyprint lang-yaml">filebeat.registry: ${path.data}/registry</pre>
+           <pre class="programlisting prettyprint lang-yaml">filebeat.registry: $\{path.data}/registry</pre>
           </div>
           <p>
            With
@@ -198,10 +198,10 @@
            :
           </p>
           <div class="pre_wrapper lang-yaml">
-           <pre class="programlisting prettyprint lang-yaml">es.host: '${ES_HOST:localhost}'
+           <pre class="programlisting prettyprint lang-yaml">es.host: '$\{ES_HOST:localhost}'
 
 output.elasticsearch:
-  hosts: ['http://${es.host}:9200']</pre>
+  hosts: ['http://$\{es.host}:9200']</pre>
           </div>
           <p>
            Introducing
html/en/beats/libbeat/master/contributing-to-beats.html
html/en/beats/libbeat/master/getting-started.html
html/en/beats/libbeat/master/index.html
html/en/beats/libbeat/master/release-highlights-7.0.0.html
html/en/beats/libbeat/master/release-highlights-7.1.0.html
html/en/beats/libbeat/master/release-highlights-7.2.0.html
html/en/beats/libbeat/master/release-highlights-7.3.0.html
html/en/beats/libbeat/master/release-highlights.html
html/en/beats/libbeat/master/release-notes-1.0.0-beta4.html
html/en/beats/libbeat/master/release-notes-1.0.0-rc1.html
html/en/beats/libbeat/master/release-notes-1.0.0-rc2.html
html/en/beats/libbeat/master/release-notes-1.0.0.html
html/en/beats/libbeat/master/release-notes-1.0.1.html
html/en/beats/libbeat/master/release-notes-1.1.0.html
html/en/beats/libbeat/master/release-notes-1.1.1.html
html/en/beats/libbeat/master/release-notes-1.1.2.html
html/en/beats/libbeat/master/release-notes-1.2.0.html
html/en/beats/libbeat/master/release-notes-1.2.1.html
html/en/beats/libbeat/master/release-notes-1.2.2.html
html/en/beats/libbeat/master/release-notes-1.2.3.html
html/en/beats/libbeat/master/release-notes-1.3.0.html
html/en/beats/libbeat/master/release-notes-1.3.1.html
html/en/beats/libbeat/master/release-notes-5.0.0-alpha1.html
html/en/beats/libbeat/master/release-notes-5.0.0-alpha2.html
--- /dev/fd/63
+++ html/en/beats/libbeat/master/release-notes-5.0.0-alpha2.html
@@ -334,7 +334,11 @@
               </a>
              </li>
              <li class="listitem">
-              Seed random number generator using crypto.rand package. ]
+              Seed random number generator using crypto.rand package.
+              <a class="ulink" href="https://github.com/elastic/beats/pull/1503{1503" target="_top">
+               https://github.com/elastic/beats/pull/1503{1503
+              </a>
+              ]
              </li>
              <li class="listitem">
               Fix beats hanging in -configtest
html/en/beats/libbeat/master/release-notes-5.0.0-alpha3.html
html/en/beats/libbeat/master/release-notes-5.0.0-alpha4.html
html/en/beats/libbeat/master/release-notes-5.0.0-alpha5.html
html/en/beats/libbeat/master/release-notes-5.0.0-beta1.html
html/en/beats/libbeat/master/release-notes-5.0.0-ga.html
html/en/beats/libbeat/master/release-notes-5.0.0-rc1.html
html/en/beats/libbeat/master/release-notes-5.0.0.html
--- /dev/fd/63
+++ html/en/beats/libbeat/master/release-notes-5.0.0.html
@@ -720,7 +720,11 @@
               </a>
              </li>
              <li class="listitem">
-              Seed random number generator using crypto.rand package. ]
+              Seed random number generator using crypto.rand package.
+              <a class="ulink" href="https://github.com/elastic/beats/pull/1503{1503" target="_top">
+               https://github.com/elastic/beats/pull/1503{1503
+              </a>
+              ]
              </li>
              <li class="listitem">
               Fix beats hanging in
html/en/beats/libbeat/master/release-notes-5.0.1.html
html/en/beats/libbeat/master/release-notes-5.0.2.html
html/en/beats/libbeat/master/release-notes-5.1.0.html
html/en/beats/libbeat/master/release-notes-5.1.1.html
html/en/beats/libbeat/master/release-notes-5.1.2.html
html/en/beats/libbeat/master/release-notes-5.2.0.html
html/en/beats/libbeat/master/release-notes-5.2.1.html
html/en/beats/libbeat/master/release-notes-5.2.2.html
html/en/beats/libbeat/master/release-notes-5.3.0.html
html/en/beats/libbeat/master/release-notes-5.3.1.html
html/en/beats/libbeat/master/release-notes-5.3.2.html
html/en/beats/libbeat/master/release-notes-5.4.0.html
html/en/beats/libbeat/master/release-notes-5.4.1.html
html/en/beats/libbeat/master/release-notes-5.4.2.html
html/en/beats/libbeat/master/release-notes-5.5.0.html
html/en/beats/libbeat/master/release-notes-5.5.1.html
html/en/beats/libbeat/master/release-notes-5.5.2.html
html/en/beats/libbeat/master/release-notes-5.5.3.html
html/en/beats/libbeat/master/release-notes-5.6.0.html
html/en/beats/libbeat/master/release-notes-5.6.1.html
html/en/beats/libbeat/master/release-notes-5.6.10.html
html/en/beats/libbeat/master/release-notes-5.6.11.html
html/en/beats/libbeat/master/release-notes-5.6.12.html
html/en/beats/libbeat/master/release-notes-5.6.13.html
html/en/beats/libbeat/master/release-notes-5.6.14.html
html/en/beats/libbeat/master/release-notes-5.6.2.html
html/en/beats/libbeat/master/release-notes-5.6.3.html
html/en/beats/libbeat/master/release-notes-5.6.4.html
html/en/beats/libbeat/master/release-notes-5.6.5.html
html/en/beats/libbeat/master/release-notes-5.6.6.html
html/en/beats/libbeat/master/release-notes-5.6.7.html
html/en/beats/libbeat/master/release-notes-5.6.8.html
html/en/beats/libbeat/master/release-notes-5.6.9.html
html/en/beats/libbeat/master/release-notes-6.0.0-alpha1.html
html/en/beats/libbeat/master/release-notes-6.0.0-alpha2.html
html/en/beats/libbeat/master/release-notes-6.0.0-beta1.html
html/en/beats/libbeat/master/release-notes-6.0.0-beta2.html
html/en/beats/libbeat/master/release-notes-6.0.0-ga.html
html/en/beats/libbeat/master/release-notes-6.0.0-rc1.html
html/en/beats/libbeat/master/release-notes-6.0.0-rc2.html
html/en/beats/libbeat/master/release-notes-6.0.0.html
html/en/beats/libbeat/master/release-notes-6.0.1.html
html/en/beats/libbeat/master/release-notes-6.1.0.html
--- /dev/fd/63
+++ html/en/beats/libbeat/master/release-notes-6.1.0.html
@@ -306,7 +306,11 @@
               </a>
              </li>
              <li class="listitem">
-              Fix logstash output debug message. ]
+              Fix logstash output debug message.
+              <a class="ulink" href="https://github.com/elastic/beats/pull/5799{5799" target="_top">
+               https://github.com/elastic/beats/pull/5799{5799
+              </a>
+              ]
              </li>
              <li class="listitem">
               Fix isolation of modules when merging local and global field settings.
html/en/beats/libbeat/master/release-notes-6.1.1.html
html/en/beats/libbeat/master/release-notes-6.1.2.html
html/en/beats/libbeat/master/release-notes-6.1.3.html
html/en/beats/libbeat/master/release-notes-6.2.0.html
html/en/beats/libbeat/master/release-notes-6.2.1.html
html/en/beats/libbeat/master/release-notes-6.2.2.html
html/en/beats/libbeat/master/release-notes-6.2.3.html
--- /dev/fd/63
+++ html/en/beats/libbeat/master/release-notes-6.2.3.html
@@ -252,6 +252,10 @@
            <div class="itemizedlist">
             <ul class="itemizedlist" type="disc">
              <li class="listitem">
+              Fix Kubernetes overview dashboard views for non default time ranges.
+              <a class="ulink" href="https://github.com/elastic/beats/issues/6395{6395}" target="_top">
+               https://github.com/elastic/beats/issues/6395{6395}
+              </a>
              </li>
             </ul>
            </div>
html/en/beats/libbeat/master/release-notes-6.3.0.html
--- /dev/fd/63
+++ html/en/beats/libbeat/master/release-notes-6.3.0.html
@@ -575,6 +575,10 @@
               </a>
              </li>
              <li class="listitem">
+              Fix Kubernetes calculated fields store.
+              <a class="ulink" href="https://github.com/elastic/beats/pull/6564{6564}" target="_top">
+               https://github.com/elastic/beats/pull/6564{6564}
+              </a>
              </li>
              <li class="listitem">
               Exclude bind mounts in fsstat and filesystem metricsets.
html/en/beats/libbeat/master/release-notes-6.3.1.html
html/en/beats/libbeat/master/release-notes-6.4.0.html
html/en/beats/libbeat/master/release-notes-6.4.1.html
--- /dev/fd/63
+++ html/en/beats/libbeat/master/release-notes-6.4.1.html
@@ -270,10 +270,7 @@
            <div class="itemizedlist">
             <ul class="itemizedlist" type="disc">
              <li class="listitem">
-              Fixed a docker input error due to the offset update bug in partial log join.
-              <a class="ulink" href="https://github.com/elastic/beats/pull/8177" target="_top">
-               8177
-              </a>
+              Fixed a docker input error due to the offset update bug in partial log join.https://github.com/elastic/beats/pull/8177[8177]
              </li>
              <li class="listitem">
               Update CRI format to support partial/full tags.
html/en/beats/libbeat/master/release-notes-6.4.2.html
html/en/beats/libbeat/master/release-notes-6.4.3.html
html/en/beats/libbeat/master/release-notes-6.5.0.html
--- /dev/fd/63
+++ html/en/beats/libbeat/master/release-notes-6.5.0.html
@@ -386,10 +386,7 @@
               </a>
              </li>
              <li class="listitem">
-              Fixed a docker input error due to the offset update bug in partial log join.
-              <a class="ulink" href="https://github.com/elastic/beats/pull/8177" target="_top">
-               8177
-              </a>
+              Fixed a docker input error due to the offset update bug in partial log join.https://github.com/elastic/beats/pull/8177[8177]
              </li>
              <li class="listitem">
               Update CRI format to support partial/full tags.
@@ -1086,10 +1083,7 @@
               <code class="literal">
                replication.master.offset
               </code>
-              .
-              <a class="ulink" href="https://github.com/elastic/beats/pull/7695" target="_top">
-               7695
-              </a>
+              .https://github.com/elastic/beats/pull/7695[7695]
              </li>
              <li class="listitem">
               Redis
html/en/beats/libbeat/master/release-notes-6.5.1.html
html/en/beats/libbeat/master/release-notes-6.5.2.html
html/en/beats/libbeat/master/release-notes-6.5.3.html
html/en/beats/libbeat/master/release-notes-6.5.4.html
html/en/beats/libbeat/master/release-notes-6.6.0.html
--- /dev/fd/63
+++ html/en/beats/libbeat/master/release-notes-6.6.0.html
@@ -398,6 +398,7 @@
               </a>
              </li>
              <li class="listitem">
+              Fix macOS default log path for elasticsearch module based on homebrew paths. {pul}8939[8939]
              </li>
              <li class="listitem">
               Support IPv6 addresses with zone id in IIS ingest pipeline.
html/en/beats/libbeat/master/release-notes-6.6.1.html
html/en/beats/libbeat/master/release-notes-6.6.2.html
html/en/beats/libbeat/master/release-notes-6.7.0.html
html/en/beats/libbeat/master/release-notes-6.7.1.html
html/en/beats/libbeat/master/release-notes-6.7.2.html
html/en/beats/libbeat/master/release-notes-6.8.0.html
html/en/beats/libbeat/master/release-notes-7.0.0-alpha1.html
--- /dev/fd/63
+++ html/en/beats/libbeat/master/release-notes-7.0.0-alpha1.html
@@ -364,7 +364,11 @@
               </a>
              </li>
              <li class="listitem">
-              Fixed Support ‘add_docker_metadata` in Windows by identifying systems’ path separator.
+              Fixed Support
+              <code class="literal">
+               add_docker_metadata
+              </code>
+              in Windows by identifying systems' path separator.
               <a class="ulink" href="https://github.com/elastic/beats/issues/7797" target="_top">
                7797
               </a>
@@ -415,6 +419,7 @@
               </a>
              </li>
              <li class="listitem">
+              Fix macOS default log path for elasticsearch module based on homebrew paths. {pul}8939[8939]
              </li>
             </ul>
            </div>
html/en/beats/libbeat/master/release-notes-7.0.0-alpha2.html
--- /dev/fd/63
+++ html/en/beats/libbeat/master/release-notes-7.0.0-alpha2.html
@@ -966,9 +966,10 @@
              </li>
              <li class="listitem">
               Add MS SQL module to X-Pack
-              <a class="ulink" href="https://github.com/elastic/beats/pull/9414[9414" target="_top">
-               https://github.com/elastic/beats/pull/9414[9414
-              </a>
+              <a class="ulink" href="https://github.com/elastic/beats/pull/9414" target="_top">
+               https://github.com/elastic/beats/pull/9414
+              </a>
+              [9414
              </li>
             </ul>
            </div>
html/en/beats/libbeat/master/release-notes-7.0.0-beta1.html
--- /dev/fd/63
+++ html/en/beats/libbeat/master/release-notes-7.0.0-beta1.html
@@ -1305,6 +1305,7 @@
               </a>
              </li>
              <li class="listitem">
+              Functionbeat can now deploy a function for Kinesis. {10116}10116[10116]
              </li>
              <li class="listitem">
               Allow functionbeat to use the keystore.
@@ -1874,6 +1875,7 @@
               </a>
              </li>
              <li class="listitem">
+              Calls to Elasticsearch X-Pack APIs made by Beats won’t cause deprecation logs in Elasticsearch logs. {9656}9656[9656]
              </li>
              <li class="listitem">
               Allow to unenroll a Beat from the UI.
html/en/beats/libbeat/master/release-notes-7.0.0-ga.html
html/en/beats/libbeat/master/release-notes-7.0.0-rc1.html
html/en/beats/libbeat/master/release-notes-7.0.0-rc2.html
html/en/beats/libbeat/master/release-notes-7.0.0.html
--- /dev/fd/63
+++ html/en/beats/libbeat/master/release-notes-7.0.0.html
@@ -1772,7 +1772,11 @@
            <div class="itemizedlist">
             <ul class="itemizedlist" type="disc">
              <li class="listitem">
-              Fix support of ‘add_docker_metadata` in Windows by identifying systems’ path separator.
+              Fix support of
+              <code class="literal">
+               add_docker_metadata
+              </code>
+              in Windows by identifying systems' path separator.
               <a class="ulink" href="https://github.com/elastic/beats/issues/7797" target="_top">
                7797
               </a>
html/en/beats/libbeat/master/release-notes-7.0.1.html
html/en/beats/libbeat/master/release-notes-7.1.0.html
html/en/beats/libbeat/master/release-notes-7.1.1.html
html/en/beats/libbeat/master/release-notes.html
html/en/beats/libbeat/master/toc.html
html/en/beats/libbeat/master/troubleshooting-upgrade.html
--- /dev/fd/63
+++ html/en/beats/libbeat/master/troubleshooting-upgrade.html
@@ -268,7 +268,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_cluster_monitor_main_is_unauthorized_for_user_literal">
+                <a id="_clustermonitormain_is_unauthorized_for_user">
                 </a>
                 <code class="literal">
                  [cluster:monitor/main] is unauthorized for user
@@ -306,7 +306,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_cluster_admin_ilm_put_is_unauthorized_for_user_literal">
+                <a id="_clusteradminilmput_is_unauthorized_for_user">
                 </a>
                 <code class="literal">
                  [cluster:admin/ilm/put] is unauthorized for user
@@ -344,7 +344,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_indices_admin_template_put_is_unauthorized_for_user_literal">
+                <a id="_indicesadmintemplateput_is_unauthorized_for_user">
                 </a>
                 <code class="literal">
                  [indices:admin/template/put] is unauthorized for user
@@ -382,7 +382,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_indices_admin_aliases_is_unauthorized_for_user_literal">
+                <a id="_indicesadminaliases_is_unauthorized_for_user">
                 </a>
                 <code class="literal">
                  [indices:admin/aliases] is unauthorized for user
@@ -420,7 +420,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_indices_data_write_bulk_is_unauthorized_for_user_literal">
+                <a id="_indicesdatawritebulk_is_unauthorized_for_user">
                 </a>
                 <code class="literal">
                  [indices:data/write/bulk] is unauthorized for user
html/en/beats/libbeat/master/upgrading-6-to-7.html
html/en/beats/libbeat/master/upgrading-minor-versions.html
html/en/beats/libbeat/master/upgrading.html
